### PR TITLE
Add member image for Myron Zhou

### PR DESCRIPTION
**Pull request by MyronZhou0**

### CODE REVIEW

**Review by claudeisakeeb — CHANGES_REQUESTED** [review]
Hi Myron, for this task you're also supposed to make changes in members.js to actually add your picture to your NPO page. Could you re-request review when those changes are pushed?

**Comment by alanchangxyz** [comment]
^ bumping, message either me or Claude if you need help finishing this up

**Comment by alanchangxyz** [comment]
can you fix merge conflicts then re-request review - you can ask one of us if you need help doing that

File: website/src/app/views/our-team/members.js [hunk 1]
```diff
@@ -457,6 +458,13 @@ const members = {
         linkedinURL: 'https://www.linkedin.com/in/michellelinCA',
         imageURL: michellelin,
       },
+      {
+        mz-member-photo
+        name: 'Myron Zhou',
+        position: 'Developer',
+        linkedinURL: 'linkedin.com/in/myron-zhou-a98270238',
```
Comment (claudeisakeeb) [inline website/src/app/views/our-team/members.js:465]: Change this to "https://..." or else it links to localhost

**Review by claudeisakeeb — CHANGES_REQUESTED** [review]
Almos there, could you fix these changes and then re-request review?

**Review by claudeisakeeb — APPROVED** [review]
Looks great, good job Myron

**Pull request merged**